addons/modules/XEH_postInit.sqf

```diff
@@ -1 +1,3 @@
-#include "script_component.hpp"
\ No newline at end of file
+#include "script_component.hpp"
+
+publicVariable QGVAR(policeStations);
```

Comment:
Should be ran only on server. Or you can set PoliceStation module to execute globally so every client will set this on his own.

Reply:
Indeed, it is wrong, I did not see this before as whole module initialization is on server side and I got stuck thinking whole `modules` component work serverside only.